src/pages/resultsView/mutation/Mutations.tsx

```diff
@@ -89,6 +89,10 @@ export default class Mutations extends React.Component<
     public render() {
         const activeTabId = this.selectedGeneSymbol;
 
+        // this is to trigger genome nexus variant annotation call as soon as the Mutations tab starts rendering
+        const triggerForAnEarlierGenomeNexusCall = this.props.store
+            .indexedVariantAnnotations.result;
```

Comment:
@alisman it also works without assigning the result to a variable. Which one do you think is better? My IDE complains about either an unused or an unassigned variable.

```suggestion
        this.props.store.indexedVariantAnnotations.result;
```

Reply:
@onursumer this is fine.